raw/src/requests/_base/errors.rs

```diff
@@ -1,16 +1,19 @@
+use failure;
+use std::result;
 use types::*;
 
-error_chain! {
-    foreign_links {
-        Json(::serde_json::Error);
-    }
+pub type Result<T, E = failure::Error> = result::Result<T, E>;
 
-    errors {
-        EmptyBody
-        TelegramError {
-            description: String,
-            parameters: Option<ResponseParameters>
-        }
-        DetachedError(err: String)
-    }
+#[derive(Debug, Fail)]
+#[fail(description = "telegram-bot-raw error")]
```

Comment:
I think I need to rename `telegram-bot-raw error` to `telegram-bot error` as these both packages use this type